patool: add page (#14785)
tldr-pages/tldr: pages/linux/patool.md

# patool

> > Patool is an archive file manager.

Reviewer: ```suggestion
> Archive file manager.
```

> > Various archive formats can be created, extracted, tested, listed, searched, repacked and compared with patool.

Reviewer: ```suggestion
> Various archive formats can be created, extracted, tested, listed, searched, repacked, and compared.
```

> More information: <https://github.com/wummel/patool>.

> - Extract an archive.:

Reviewer: ```suggestion
- Extract an archive:
```

> `patool extract $ARCHIVE`

Reviewer: ```suggestion
`patool extract {{path/to/archive}}`
```

- List contents of an archive:

> `patool list $ARCHIVE`

Reviewer: ```suggestion
`patool list {{path/to/archive}}`
```

> - Run diff on contents of two archives and print to stdout:

Reviewer: ```suggestion
- Compare the contents of two archives and display the differences in the standard output:
```

> `patool diff $ARCHIVE1 $ARCHIVE2`

Reviewer: ```suggestion
`patool diff {{path/to/archive1}} {{path/to/archive2}}`
```

- Search for a string inside the contents of an archive:

> `patool search $ARCHIVE`

Reviewer: ```suggestion
`patool search {{path/to/archive}}`
```
